### ISSUE TITLE
F1TV | F1TV.com

### ISSUE THREAD

**Issue by hgt27** [PreMiD/Presences#3360]
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [x] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?**
lepfla#2414 

**What's the name of the service?*
https://f1tv.formula1.com

**What should the Presence display?**
the presence should display the content as with the Netflix presence or the Youtube presence, 

basically, the presence should display the category you are looking at and the video or live you are looking at with the time spent when looking at a live or video.  

**If possible, please provide a logo for the service (512x512 minimum)**
![410yMUyZaVL](https://user-images.githubusercontent.com/71899602/112722791-c1934b00-8f0b-11eb-94f6-e4208d4c862f.png)
